openwebstart/src/main/java/com/openwebstart/app/ui/ApplicationManagerPanel.java

```diff
@@ -74,7 +72,7 @@ public ApplicationManagerPanel(final DeploymentConfiguration deploymentConfigura
 
     private void refreshModel() {
         listModel.clear();
-        backgroundExecutor.execute(() -> {
+        getDaemonExecutorService().execute(() -> {
```

Comment:
It won't be a problem to have this one as a non daemon executor. I prefer to change as less behaviour as possible with this PR so we should change it here to non daemon.